packages/nextjs/app/builders/[address]/page.tsx

```diff
@@ -13,7 +14,17 @@ export default async function BuilderPage({ params }: { params: { address: strin
   const bgMemberExists = await isBgMember(userAddress);
 
   if (!user) {
-    return <div>User not found</div>;
+    return (
+      <NotFound
+        heading="User Not Found"
+        extraElement={
+          <>
+            <p className="mb-0">Double check the Ethereum address is correct in the URL</p>
+            <p className="m-0">Make sure not to use an ENS address in the URL</p>
+          </>
+        }
+      />
+    );
```

Comment:
The problem with this approach is that we are getting a 200 (this is bad for indexing)

![image](https://github.com/user-attachments/assets/8f544238-af55-4e0b-8905-3ac666cc5747)

Is it possible to return a 404 with a customized page? (as `notFound()`)

Reply:
Ah yes, I added a new `not-found.tsx` inside of the `/builders/[address]` folder. This will make it custom for this route

Reply:
Great stuff!